CONTRIBUTING.md

```diff
@@ -248,6 +248,84 @@ Optional `--namespace` and `--mode` parameters can be used to configure differen
 
 With the configuration in place, you can launch the MCP server directly from your IDE or any tooling that uses `mcp.json`.
 
+### Configuring External MCP Servers
+
+The Azure MCP Server supports connecting to external MCP servers through an embedded `registry.json` configuration file. This enables the server to act as a proxy, aggregating tools from multiple MCP servers into a single interface. The registry follows the same configuration schema as VS Code's `mcp.json`.
+
+#### Registry Configuration
+
+External MCP servers are defined in the embedded resource file `src/Areas/Server/Resources/registry.json`. This file contains server configurations that support both SSE (Server-Sent Events) and stdio transport mechanisms, following the standard MCP configuration format.
+
+The registry structure follows this format:
+
+```json
+{
+  "servers": {
+    "server-id": {
+      "url": "https://example.com/api/mcp",
+      "description": "Description of what this server provides"
+    },
+    "another-server": {
+      "type": "stdio",
+      "command": "path/to/executable",
+      "args": ["arg1", "arg2"],
+      "env": {
+        "ENV_VAR": "value"
+      },
+      "description": "Another MCP server using stdio transport"
```

Comment:
non-blocking suggestion: you could use the values from https://github.com/microsoftdocs/mcp as a real example.